src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -121,17 +121,22 @@ class RelayDefaultNetworkLayer {
         }
       }
       init = {
+        ...this._init,
         body: formData,
         method: 'POST',
       };
     } else {
       init = {
+        credentials: 'same-origin',
```

Comment:
I like it since it makes `fetch` act more like a regular `XMLHttpRequest`. That being said, since it's easily configurable now, I could swing either way.